exercises/5.1/solution_ws/ros2/src/lesson_perception/package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>lesson_perception</name>
+  <version>0.0.1</version>
+  <description>
+     The lesson_perception package
+  </description>
+
+  <maintainer email="tiffany.cappellari@swri.org">tcappellari</maintainer> -->
```

Comment:
Might be nice to include `<author>` tags for the people who wrote the ROS1 versions - this code is heavily based on their work.